src/Model/Ticket.java

```diff
@@ -1,81 +1,73 @@
 package Model;
 
+import Model.Enums.ConcertHall;
+import Model.Enums.StadiumSector;
+import View.Printable;
+
 import java.math.BigDecimal;
 import java.sql.Timestamp;
 import java.text.DecimalFormat;
-import java.text.ParseException;
 import java.text.SimpleDateFormat;
 import java.time.LocalDateTime;
-import java.time.chrono.ChronoLocalDateTime;
 import java.time.format.DateTimeFormatter;
-import java.time.format.DateTimeFormatterBuilder;
-import java.time.format.FormatStyle;
 
-public class Ticket {
-    private static int ticketsCounter = 1;
-    private final LocalDateTime ticketCreationTime = LocalDateTime.now();
+public class Ticket implements ID, Printable {
```

Comment:
the same, can't find interface ID. Perhaps you replaced it by Entity abstract class

Reply:
Fixed. I replaced using ID declaration with extending abstract class Entity